passenger-service/src/main/java/com/modsen/passengerservice/service/impl/PassengerServiceImpl.java

```diff
@@ -0,0 +1,82 @@
+package com.modsen.passengerservice.service.impl;
+
+import com.modsen.passengerservice.dto.PassengerDTO;
+import com.modsen.passengerservice.dto.PassengerListDTO;
+import com.modsen.passengerservice.entity.Passenger;
+import com.modsen.passengerservice.exception.PassengerNotFoundException;
+import com.modsen.passengerservice.mapper.PassengerMapper;
+import com.modsen.passengerservice.repository.PassengerRepository;
+import com.modsen.passengerservice.service.PassengerService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class PassengerServiceImpl implements PassengerService {
+
+    private final PassengerRepository passengerRepository;
+    private final PassengerMapper passengerMapper;
+
+    @Override
+    public PassengerListDTO getAllPassengers() {
+        log.info("Retrieving all passengers");
+        List<PassengerDTO> passengers = passengerMapper.toListDTO(passengerRepository.findAll());
```

Comment:
Лучше разнести это отдельно на вытягивание из бд и маппинг, так код будет более читаемым
```suggestion
        List<Passenger> passengers = passengerRepository.findAll();
        List<PassengerDTO> passengersDto = passengerMapper.toListDTO(passengers);
```